### PR TITLE
Remove CCM dependency pkg/util/flag

### PR DESCRIPTION
**Pull request by cici37**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug

/kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

This PR is part of the issue [here](https://github.com/kubernetes/cloud-provider/issues/29). See [KEP-removing-in-tree-providers](https://github.com/kubernetes/enhancements/blob/master/keps/sig-cloud-provider/20190125-removing-in-tree-providers.md) for more details.
This PR is to remove CCM dependency pkg/util/flag which moves printFlag func under k8s.io/component-base/cli/flag.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/util/flag/flags.go [hunk 1]
```diff
@@ -22,18 +22,10 @@ import (
 	"strconv"
 
 	"github.com/spf13/pflag"
-	"k8s.io/klog"
 
 	utilnet "k8s.io/apimachinery/pkg/util/net"
 )
 
-// PrintFlags logs the flags in the flagset
-func PrintFlags(flags *pflag.FlagSet) {
```
Comment (andrewsykim) [inline pkg/util/flag/flags.go:31]: This seems like an exported function that could have a lot of external consumers. Wondering if we should leave this copy here to be as less disruptive to those consumers but add a comment pointing to the new preferred location.

Comment (cici37) [reply pkg/util/flag/flags.go:31]: Is there a way to verify if there are external consumers for this function?

Comment (cici37) [reply pkg/util/flag/flags.go:31]: Is that possible to handle this concern by adding the switch to some documentation and maybe leave comments in code?

Comment (andrewsykim) [reply pkg/util/flag/flags.go:31]: I'll defer to @liggitt on this one. I don't think it hurts to leave the extra copy here with comments that link to the new location but maybe there aren't as many consumers of this than I initially thoughts.

Comment (lavalamp) [reply pkg/util/flag/flags.go:31]: I think people shouldn't be directly importing this package. We discourage importing things not published in one of the staging repos.

**Review by andrewsykim — COMMENTED** [review]
/lgtm

**Pull request merged**